bundles/ranvier-combat/behaviors/item/decay.js

```diff
@@ -37,8 +37,12 @@ module.exports = srcPath => {
         }
 
         if (belongsTo) {
+          const owner = this.findOwner();
+          if (owner) {
+            Broadcast.sayAt(owner, `Your ${this.name} has rotted away!`);
+            Broadcast.prompt(owner);
```

Comment:
Yeah I've generally been following the model I mentioned it makes the game a lot less spammy. Especially if you're just doing stuff like chatting